### PR TITLE
PSMDB-426: Update for v4.2.0-1

### PR DESCRIPTION
**Pull request by borys-belinsky**

### CODE REVIEW

File: source/install/apt.rst [hunk 1]
```diff
@@ -9,6 +9,7 @@ of the following distributions:
 
 * Debian 8 (jessie)
 * Debian 9 (stretch)
+* Debian 10 (buster)
 * Ubuntu 14.04 LTS (Trusty Tahr)
```
Comment (tplavcic) [inline source/install/apt.rst]: `Debian 8 (jessie)` and `Ubuntu 14.04 LTS (Trusty Tahr)` not available any more

File: source/install/tarball.rst [hunk 2]
```diff
@@ -11,17 +11,17 @@ You can find links to the binary tarballs from the `Percona website`_
 
    .. code-block:: bash
 
-      $ wget https://www.percona.com/downloads/percona-server-mongodb-4.0/\
-      percona-server-mongodb-4.0.4-1/binary/debian/jessie/x86_64/\
-      percona-server-mongodb-4.0.4-1-ra1cb178-jessie-x86_64-bundle.tar
-      $ tar xfz percona-server-mongodb-4.0.4-1-ra1cb178-jessie-x86_64-bundle.tar
+      $ wget https://www.percona.com/downloads/percona-server-mongodb-4.2/\
+      percona-server-mongodb-4.2.0-1/binary/debian/jessie/x86_64/\
```
Comment (tplavcic) [inline source/install/tarball.rst]: debian jessie binary not available, maybe put one from ubuntu bionic instead: percona-server-mongodb-4.2.0-1-bionic-x86_64.tar.gz
https://jenkins.percona.com/job/percona-server-mongodb-4.2-TARBALL-release/label_exp=min-bionic-x64/

File: source/install/upgrade_from_40.rst [hunk 3]
```diff
@@ -34,7 +34,7 @@ Upgrading on Debian or Ubuntu
 |tip.run-all.root|
 
 1. Stop the ``mongod`` instance: :bash:`service mongod stop`
-#. Remove |PSMDB| |prev-version| packages: :bash:`apt-get remove percona-server-mongodb-36*`
+#. Remove |PSMDB| |prev-version| packages: :bash:`apt-get remove percona-server-mongodb-40*`
```
Comment (tplavcic) [inline source/install/upgrade_from_40.rst]: 4.0 packages do not have major version in their name (this change happened in 4.0 line), so `percona-server-mongodb-40*` should be `percona-server-mongodb*`

File: source/install/upgrade_from_40.rst [hunk 4]
```diff
@@ -46,7 +46,7 @@ Upgrading on RHEL and CentOS
 |tip.run-all.root|
 
 1. Stop the ``mongod`` instance: :bash:`service mongod stop`
-#. Remove |PSMDB| |prev-version| packages: :bash:`yum remove Percona-Server-MongoDB-36*`
+#. Remove |PSMDB| |prev-version| packages: :bash:`yum remove Percona-Server-MongoDB-40*`
```
Comment (tplavcic) [inline source/install/upgrade_from_40.rst]: same for rpm (no 40 in package name), but for rpm they are also all lowercase, so `Percona-Server-MongoDB-40*` should be `percona-server-mongodb*`

**Review by tplavcic — CHANGES_REQUESTED** [review]

File: source/install/tarball.rst (hunk 2 above)
Comment (tplavcic) [reply source/install/tarball.rst]: First two lines still have Debian Jessie binary tarball link if I'm not missing something.

**Review by tplavcic — CHANGES_REQUESTED** [review]

File: source/install/yum.rst [hunk 5]
```diff
@@ -43,11 +43,13 @@ Package Contents
 Installing from Percona Repositories
 ================================================================================
 
-It is recommended to intall |PSMDB| from official Percona repositories:
+It is recommended to intall |PSMDB| from official |percona| repositories:
```
Comment (igorsol) [inline source/install/yum.rst]: Small typo in the word `intall`

**Review by igorsol — APPROVED** [review]

**Review by tplavcic — APPROVED** [review]

**Pull request merged**